numba/ir_utils.py

```diff
@@ -1682,3 +1682,44 @@ def is_namedtuple_class(c):
     if not isinstance(fields, tuple):
         return False
     return all(isinstance(f, str) for f in fields)
+
+def raise_on_unsupported_feature(func_ir):
+    """
+    Helper function to walk IR and raise if it finds op codes
+    that are unsupported. Could be extended to cover IR sequences
+    as well as op codes. Intended use is to call it as a pipeline
+    stage just prior to lowering to prevent LoweringErrors for known
+    unsupported features.
+    """
+    for blk in func_ir.blocks.values():
+        for stmt in blk.body:
```

Comment:
this loop can be simplified with `blk.find_exprs()`

Reply:
Yes, thanks, I always for get that! Fixed.